wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -779,6 +779,41 @@ def icon(name=None, classname=None, title=None, wrapped=False, class_name=None):
     }
 
 
+@register.inclusion_tag("wagtailadmin/shared/status_tag.html")
+def status(
+    label=None,
+    classname=None,
+    url=None,
+    title=None,
+    hidden_label=None,
+    attrs=None,
```

Comment:
@salty-ivy  adding a trailing comma on longer lists (e.g. params) will make the formatter (black) wrap to multiple lines. There is no hard rule but after about 3 or 4 items it reads better if they are all on their own line.